src/PythonMigrationViewExtension/PythonMigrationAssistantViewModel.cs

```diff
@@ -10,11 +12,13 @@ internal class PythonMigrationAssistantViewModel
         public string OldCode { get; set; }
         public string NewCode { get; set; }
         private PythonNode PythonNode { get; set; }
+        public DynamoViewModel DynamoViewModel { get; set; }
 
-        public PythonMigrationAssistantViewModel(PythonNode pythonNode)
+        public PythonMigrationAssistantViewModel(PythonNode pythonNode, DynamoViewModel dynamoViewModel)
```

Comment:
can we depend on something less hard to get than the `DynamoViewModel` ? Extension access to that is going away eventually.

I think there's a way to get the `backup directory` path without the full viewmode, there's some static settings providers or things like that from what i remember.

Reply:
as far as i can see i can only get it from the `PathManager` which i can only get from the `DynamoModel`. I havent been able to find any other place to get it from, but can have another look.

Im also using the `SaveAs` on the `DynamoModel` which is the same as what Dynamo uses to save the regular backups.